src/ast.scm

```diff
@@ -338,40 +342,47 @@
            ((meta)  ;; allow certain per-argument annotations
             (if (nospecialize-meta? v #t)
                 (arg-name (caddr v))
-                (bad-formal-argument v)))
+                (bad-formal-argument v fname)))
            ((kw)
-            (arg-name (cadr v)))
-           (else (bad-formal-argument v))))))
+            (arg-name (cadr v) fname))
+           (else (bad-formal-argument v fname))))))
+
+(define (arg-name-🍛 fname)
```

Comment:
:laughing:

Reply:
Yeah, couldn't help myself with the pun :)